### PR TITLE
Bot API 7.5

### PR DESCRIPTION
**Pull request by coder2020official**
Bot API 7.5

- [x] Added the classes [StarTransactions](https://core.telegram.org/bots/api#startransactions), [StarTransaction](https://core.telegram.org/bots/api#startransaction), [TransactionPartner](https://core.telegram.org/bots/api#transactionpartner) and [RevenueWithdrawalState](https://core.telegram.org/bots/api#revenuewithdrawalstate), containing information about Telegram Star transactions involving the bot.
- [x] Added the method [getStarTransactions](https://core.telegram.org/bots/api#getstartransactions) that can be used to get the list of all Telegram Star transactions for the bot.
- [x] Added support for callback buttons in [InlineKeyboardMarkup](https://core.telegram.org/bots/api#inlinekeyboardmarkup) for messages sent on behalf of a business account.
- [x] Added support for callback queries originating from a message sent on behalf of a business account.
- [x] Added the parameter business_connection_id to the methods [editMessageText](https://core.telegram.org/bots/api#editmessagetext), [editMessageMedia](https://core.telegram.org/bots/api#editmessagemedia), [editMessageCaption](https://core.telegram.org/bots/api#editmessagecaption), [editMessageLiveLocation](https://core.telegram.org/bots/api#editmessagelivelocation), [stopMessageLiveLocation](https://core.telegram.org/bots/api#stopmessagelivelocation) and [editMessageReplyMarkup](https://core.telegram.org/bots/api#editmessagereplymarkup), allowing the bot to edit business messages.
- [x] Added the parameter business_connection_id to the method [stopPoll](https://core.telegram.org/bots/api#stoppoll), allowing the bot to stop polls it sent on behalf of a business account.

### CODE REVIEW

**Comment by coder2020official** [comment]
@Badiboy done, need your review

**Comment by Badiboy** [comment]
Got it

File: telebot/types.py [hunk 1]
```diff
@@ -10076,3 +10076,245 @@ def de_json(cls, json_string):
...
+
+    :return: Instance of the class
+    :rtype: :class:`RevenueWithdrawalStateFailed`
+    """
+
+    def __init__(self, type, **kwargs):
+        self.type: str = type
+
+
+class TransactionPartner(JsonDeserializable):
```
Comment (Badiboy) [inline telebot/types.py:10169]: The same as for RevenueWithdrawalState - de_json for Childs

File: telebot/types.py [hunk 2]
```diff
@@ -10076,3 +10076,245 @@ def de_json(cls, json_string):
...
+    :param user: Information about the user
+    :type user: :class:`User`
+
+    :return: Instance of the class
+    :rtype: :class:`TransactionPartnerUser`
+    """
+
+    def __init__(self, type, user, **kwargs):
+        self.type: str = type
+        self.user: User = user
```
Comment (Badiboy) [inline telebot/types.py:10236]: We need do de_json USER somewhere (in de_json of class, I suppose).

**Comment by coder2020official** [comment]
If I understood you right - I just forgot all de_json methods for classes

**Comment by coder2020official** [comment]
btw, I'm done just in case you missed @Badiboy

**Comment by Badiboy** [comment]
LGTM, thank you.

**Pull request merged**